src/plugins/ui_feature/format.ts

```diff
@@ -43,7 +46,7 @@ export class FormatPlugin extends UIPlugin {
         for (let row = zone.top; row <= zone.bottom; row++) {
           const position = { sheetId, col, row };
           const pivotCell = this.getters.getPivotCellFromPosition(position);
-          if (pivotCell.type === "VALUE") {
+          if (this.isBasicPivotValueFormula(position, pivotCell)) {
```

Comment:
TBH I would just disable the functionnality on non spreaded pivot formula. It doens't make much sense for splitted formulas to me and sounds very much like black magic that people will not really understand. 

I don't expect changing the format on a single =PIVOT.VALUE cell that I put alone on a random sheet to affect the format of the whole pivot on all the other sheet. The point of having splitted pivot is that I can do things on it independantly from the pivot no ?